jsettlers.common/resources/localization/labels_en.properties

```diff
@@ -147,45 +148,45 @@ controlpanel_weapons_title = Weapons
 controlpanel_distribution_title = Goods Distribution
 
 # Settlers Control Panel Titles
-warriors_title = Warriors
+warriors_title = Soldiers
 
 # warriors panel
-upgrade_warriors = upgrade %s
-upgrade_warriros_progress = godly upgrade: %d%%
-combat_strength = %d%% combat strength
+upgrade_warriors = Promote %s
+upgrade_warriros_progress = Divine Promotion: %d%%
+combat_strength = Strength: %d%%
 
 # settler selection
 headline_materials = materials
 headline_movabletypes = types
 
 # gui labels
-under_construction = %s (being built)
-materials_required = materials required
-productivity = productivity: %d%%
+under_construction = %s (Being built)
+materials_required = Materials Needed:
+productivity = Productivity: %d%%
 click_set_workcenter = Click to set the work area for this building.
-click_to_move = Click to sent this settler.
-abort = abort
-really_destroy_building = Are you sure you want to raze this building?
+click_to_move = Click to move this settler.
+abort = cancel
+really_destroy_building = Are you sure you want to destroy this building?
 priority_HIGH = Set priority to high.
 priority_LOW = Set priority to normal.
 priority_STOPPED = Stop working
 building-build-in-progress = Construction site for: %s
 building-material-count = %d
 building-material-required = %d / %d
-stop = stop
```

Comment:
Currently, our stop is not the same as hold position. Geologists will just stand around, but soldiers will just stop walking but still attack (and walk away for attacking).